src/coach/consumer.py

```diff
@@ -0,0 +1,60 @@
+import json
+import pika
+from .fitness_coach_service import request_wod
+import logging
+import os
+
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logging.getLogger("pika").setLevel(logging.DEBUG)
+
+
+QUEUE_NAME = "createWodQueue"
+
+def callback(ch, method, properties, body):
+    logging.info(f"[Consumer] Received message: {body}")
+    try:
+        data = json.loads(body)
+        user_email = data.get("user_email")
```

Comment:
Here we don't use defined message format for queue communication. If the message changes we won't notice it before deploying in production